corehq/apps/app_manager/static/app_manager/js/summary/case_summary.js

```diff
@@ -1,7 +1,13 @@
-hqDefine('app_manager/js/summary/case_summary', function() {
-    var assertProperties = hqImport("hqwebapp/js/assert_properties"),
-        initialPageData = hqImport("hqwebapp/js/initial_page_data"),
-        models = hqImport("app_manager/js/summary/models");
+// knockout_bindings.ko is a part of this dependency because page uses 'slideVisible' binding in case_summary.html
+hqDefine('app_manager/js/summary/case_summary',[
```

Comment:
@orangejenny do you know if our JS linter doesn't try to apply the "single space between function arguments" style rule? I thought it would, so just want to make sure this isn't a sign that the linter/sticklerbot is broken or something.

Reply:
We don't have comma-spacing turned on. Most of the whitespace rules aren't part of the default eslint set, probably because they're stylistic, and we haven't added many to our config. I'd be open to using whitespace rules to enforce consistency, feel free to add.